### PR TITLE
Updated PV alerts

### PR DESCRIPTION
**Pull request by jewzaam**
https://issues.redhat.com/browse/OSD-3085

Additional alerts for customers:
* KubePersistentVolumeUsageCriticalCustomer
* KubePersistentVolumeFullInFourDaysCustomer

Additional alerts for non-RHMI layered products:
* KubePersistentVolumeUsageCriticalLayeredProduct
* KubePersistentVolumeFullInFourDaysLayeredProduct

SOPs need created / updated.

### CODE REVIEW

File: deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      # This matches with the default KubePersistentVolumeUsageCritical; this rule inverts the regex match.
+    - alert: KubePersistentVolumeUsageCriticalCustomer
+      expr: |
+        100
+        * kubelet_volume_stats_available_bytes{job="kubelet",namespace!~"(^openshift-.*|^kube-.*|^default$|^redhat-.*)"}
+        / kubelet_volume_stats_capacity_bytes{job="kubelet",namespace!~"(^openshift-.*|^kube-.*|^default$|^redhat-.*)"}
+        < 3
+      labels:
+        severity: critical
+        namespace: dedicated-admin
```
Comment (cblecker) [inline deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml]: Why are we overriding the label here? Shouldn't this come from the PV that is having high usage? (same question for all alerts)

**Comment by cblecker** [comment]
/hold
for review

File: deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml (hunk 1 above)
Comment (jewzaam) [reply deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml]: Initially thought we didn't get a namespace in the result to work with but in poking it again I see we do.  I'll update it.

**Comment by cblecker** [comment]
@jewzaam Needs another `make`

**Comment by jewzaam** [comment]
@cblecker thanks, pushed

**Pull request merged**